patches/chrome-installer-linux-debian-update_dist_package_versions.py.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/chrome/installer/linux/debian/update_dist_package_versions.py b/chrome/installer/linux/debian/update_dist_package_versions.py
+index 481dfd7b1930385ff44862dc9cd0ba112b422417..ae2283c515e9ae7c2ee9f8bb5c276a5204de7e04 100755
+--- a/chrome/installer/linux/debian/update_dist_package_versions.py
++++ b/chrome/installer/linux/debian/update_dist_package_versions.py
+@@ -40,6 +40,7 @@ PACKAGE_FILTER = set([
+     "libatk-bridge2.0-0",
+     "libc6",
+     "libcairo2",
++    "libcairo-gobject2",
```

Comment:
I think we'll be less likely to run into conflicts if we put these at the top of the list instead of keeping the alphabetical order - same with the other file